### PR TITLE
Feat/add analytics

### PR DESCRIPTION
**Pull request by fricoben**
This add analytics on some buttons and on page views. 

I'm not sure it adds analytics with IPs and unique users but I asked to the client support

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -29,7 +61,38 @@ function MyApp({ Component, pageProps }: AppProps) {
               />
             </Head>
             <Navbar />
-            <Component {...pageProps} />
+            <Notification visible={!cookiesAccepted} severity="info">
+              <div className="flex flex-wrap sm:gap-20 gap-5">
+                <p>
+                  We use cookies to ensure you get the best experience on our
```
Comment (Th0rgal) [inline pages/_app.tsx]: what cookies do we use?

File: pages/_app.tsx [hunk 2]
```diff
@@ -29,7 +61,38 @@ function MyApp({ Component, pageProps }: AppProps) {
               />
             </Head>
             <Navbar />
-            <Component {...pageProps} />
+            <Notification visible={!cookiesAccepted} severity="info">
```
Comment (Th0rgal) [inline pages/_app.tsx]: I think it would look cleaner as a separate component instead of something within _app.tsx

**Review by Th0rgal — CHANGES_REQUESTED** [review]

File: pages/_app.tsx (hunk 2 above)
Comment (fricoben) [reply pages/_app.tsx]: We want this to be everywhere on the app if the guy didn't accept it, I don't see any other solutions

File: pages/_app.tsx (hunk 1 above)
Comment (fricoben) [reply pages/_app.tsx]: No cookies just gather data I guess, What do you propose ?

File: pages/_app.tsx (hunk 2 above)
Comment (Th0rgal) [reply pages/_app.tsx]: I meant in the code, you could create a <PrivacyNotification /> component and import it

File: pages/_app.tsx (hunk 1 above)
Comment (Th0rgal) [reply pages/_app.tsx]: just asking them to accept the privacy policy should be enough then :)

File: pages/_app.tsx (hunk 2 above)
Comment (fricoben) [reply pages/_app.tsx]: Ok I see

**Review by Th0rgal — APPROVED** [review]
lgtm

**Pull request merged**